### ISSUE TITLE
PHP7.1 Warning: A non-numeric value encountered lib-common.php line 8685

### ISSUE THREAD

**Issue by ivywe** [Geeklog-Core/geeklog#814]
PHP7.1 Warning: A non-numeric value encountered lib-common.php line 8685

from:
```
    if (($_VARS['last_scheduled_run']
```
 to:
```
    if (((int)$_VARS['last_scheduled_run']
```

**Comment by eSilverStrike** [comment]
We should check why as well. Maybe the initial value is an empty field so the above fix would be fine.

**Comment by mystralkk** [comment]
This bug should have been fixed by @eSilverStrike with Geeklog 2.1.3.  @ivywe , what version are you using?

**Comment by ivywe** [comment]
I(m using 2.1.3 
https://github.com/ivywe/geeklog-ivywe

When I set Geeklog CRON set in configuration, the error occurred in the top page.

**Comment by mystralkk** [comment]
The lines 8684-8696 of lib-common.php shipped with Geeklog 2.1.3 read as follows:

`
// Check and see if any plugins (or custom functions)
// have scheduled tasks to perform
if (!isset($_VARS['last_scheduled_run']) OR !is_numeric($_VARS['last_scheduled_run'])) {
    $_VARS['last_scheduled_run'] = 0;
}
if ($_CONF['cron_schedule_interval'] > 0 && COM_onFrontpage()) {
    if (($_VARS['last_scheduled_run']
            + $_CONF['cron_schedule_interval']) <= time()
    ) {
        DB_query("UPDATE {$_TABLES['vars']} SET value=UNIX_TIMESTAMP() WHERE name='last_scheduled_run'");
        PLG_runScheduledTask();
    }
}
`

On the other hand, the same lines of lib-common.php in your repository read:

`
    if (($_VARS['last_scheduled_run']
            + $_CONF['cron_schedule_interval']) <= time()
    ) {
        DB_query("UPDATE {$_TABLES['vars']} SET value=UNIX_TIMESTAMP() WHERE name='last_scheduled_run'");
        PLG_runScheduledTask();
    }
}
define('GL_INITIALIZED', true);
`

It is obvious that your repository hasn't synced with https://github.com/Geeklog-Core/geeklog/ .

**Comment by eSilverStrike** [comment]
So I'll close this issue then since the main repository is fine.
